src/fides/api/service/connectors/saas/connector_registry_service.py

```diff
@@ -147,11 +160,23 @@ def _register_template(
         Registers a custom connector template by converting it to a ConnectorTemplate
         and adding it to the loader's template dictionary.
         """
+
+        config = SaaSConfig(**load_config_from_string(template.config))
+        authentication = config.client_config.authentication
+        authorization_required = (
+            authentication.strategy
```

Comment:
This was a bit hard for me to read because it's doing a nested comparison instead of using `and`, I think something like this might be a bit more readable:

```
In [1]: foo = None

In [2]: bar = True if foo and foo.test else False
```

Reply:
Good call!